src/ot/layout/common.rs

```diff
@@ -0,0 +1,835 @@
+//! Common tables for OpenType layout.
+
+use std::cmp::Ordering;
+use std::convert::TryFrom;
+
+use ttf_parser::parser::{
+    FromData, LazyArray16, LazyArray32, Offset, Offset16, Offset32, Offsets16, Stream, TryNumFrom,
+};
+use ttf_parser::GlyphId;
+
+use crate::font::Font;
+use crate::Tag;
+
+/// A GSUB or GPOS table.
+#[derive(Clone, Copy, Debug)]
+pub struct SubstPosTable<'a> {
+    scripts: RecordList<'a>,
+    features: RecordList<'a>,
+    lookups: LookupList<'a>,
+    variations: Option<FeatureVariations<'a>>,
+}
+
+#[allow(dead_code)]
```

Comment:
Still dead?

Reply:
Well, yes, some of the functions. Scripts and features have an analogous set of four functions and not all of them are used. I thought it would be nice to have them for consistency, but we could also remove them.

Reply:
I think it would be better to remove unused code.

Reply:
Done.